src/components/map/MapService.js

```diff
@@ -1530,20 +1530,39 @@ goog.require('ga_urlutils_service');
           return layer;
         },
 
+        flyToAnimation: function(ol3d, center, destination, defer) {
+          var scene = ol3d.getCesiumScene();
+          var camera = scene.camera;
+          var globe = scene.globe;
+          var carto = globe.ellipsoid.cartesianToCartographic(destination);
+          $http.get(gaGlobalOptions.apiUrl + '/rest/services/height', {
```

Comment:
@oterral, you can use the sample terrain call. It will work but the drawback is that it will download a full tile (at level 17 since it is what you asked for).